sky/global_user_state.py

```diff
@@ -663,6 +664,23 @@ def get_enabled_clouds() -> List[clouds.Cloud]:
     return enabled_clouds
 
 
+def get_enabled_storage_clouds() -> List[str]:
+    # This is a temporary solution until https://github.com/skypilot-org/skypilot/issues/1943 # pylint: disable=line-too-long
+    # is resolved by implementing separate 'enabled_storage_clouds'
+    enabled_clouds = get_enabled_clouds()
+    # TODO: Remove clouds.IBM from below condition as IBM COS gets integrated
+    enabled_storage_clouds = [
+        str(cloud)
+        for cloud in enabled_clouds
+        if not isinstance(cloud, clouds.Lambda) and
+        not isinstance(cloud, clouds.IBM)
```

Comment:
Can we:
1. Update `STORE_ENABLED_CLOUDS` in storage.py to a) be a List[str] and b) Add cloudflare.NAME to this list?
2. Change this list comprehension to select only the clouds in `STORE_ENABLED_CLOUDS`?

Trying to minimize the number of places the list of storage enabled clouds needs to be maintained :)

Reply:
Updated.
Note the change I made in `task.py/get_preferred_store_type()` as `STORE_ENABLED_CLOUDS` is `List[str]` now.